arangodb-net-standard/Serialization/ApiClientSerializationOptions.cs

```diff
@@ -16,6 +16,11 @@ public class ApiClientSerializationOptions
         /// </summary>
         public bool IgnoreNullValues { get; set; }
 
+        /// <summary>
+        /// True to ignore missing members, otherwise false.
```

Comment:
Should we mention "to ignore missing members when deserializing" or do you think it's obvious enough?

Reply:
Thanks for the suggesttion. I have added it.